docs/versioned_docs/version-v1.9/concepts/before-oauth2.mdx

```diff
@@ -33,7 +33,7 @@ If you use an API that works with OAuth2 or OpenID Connect, then of course you
 need to use that API - but then you are on the OAuth2 / OpenID Connect client
 side!
 
-If you do not have user registration, password recovery, login form, session
+If you have user registration, password recovery, login form, session
 management, or user settings system in place then OAuth2 is most likely not the
 right fit for you. If that's the case, check out
```

Comment:
The idea of this is portraying:

If you have nothing that solves registration, password recovery, ... and you still need to build that - then you probably don't need OAuth2 **yet** because you have to solve these things first. You can add OAuth2 later, if you want, but before doing that check out ORY Kratos.

Maybe we can improve the wording for this, do you have ideas?

Reply:
Maybe: 
`
If you want a system for user registration, password recovery, login form, session management and user settings, then don't start with OAuth2. It is recommended to use [ORY Kratos](https://www.ory.sh/kratos) for your basic user management needs and then figure out if you still need OAuth2.`

The whole sentence is kind of a repeat of the :::note:::, but I think its a very important point to really hammer home, so does not hurt to repeat imo.

Reply:
Would you be down to update that @vinckr ? :)